dotcom-rendering/src/web/components/Card/components/CardWrapper.tsx

```diff
@@ -99,14 +103,14 @@ export const CardWrapper = ({
 	format,
 	containerPalette,
 	containerType,
-	transparent,
+	isDynamo,
 }: Props) => {
 	const palette = decidePalette(format, containerPalette);
 	return (
 		<div
 			css={[
-				cardStyles(format, palette, containerType),
-				transparent &&
+				cardStyles(format, palette, containerType, isDynamo),
+				isDynamo &&
 					css`
 						background: transparent;
 					`,
```

Comment:
If both of these styles are dependent on `isDynamo`, maybe the transparent background can be folded in `cardStyles` for consistency?

Reply:
Great spot!